### ISSUE TITLE
Any support or suggestion of initializing a ML repo with your codes locally and create related pipelines?

### ISSUE THREAD

**Issue by jianawu** [Azure/mlops-v2#122]
I am trying to start up a project with mlops. The initialized ml repo with pipelines of classical type is really what I want.
However, I am not able to have enough permissions to follow the deployment guidelines to import all the repos into my organization or grant permissions to let any repo create pipelines automatically within my organization. 

Are we able to provide some guidelines to help users to initial the repo locally through a cli? like clone the repos to the disk, run scripts to create draft repo locally and then give guidelines to help users create related pipelines?

At the same time, would you like to give me some suggestions in this post to let me know the best practice now to initialize a ML repo quickly in local? Try to read your deploy scripts and manually trigger part of them is the only way now?

Thanks for your help!

**Comment by jianawu** [comment]
Also, if there are sample ML repos with pipelines. It will also be very helpful to me to learn from.

**Comment by setuc** [comment]
@jianawu  Did you take a look at this: https://learn.microsoft.com/en-us/azure/machine-learning/how-to-setup-mlops-azureml?view=azureml-api-2&tabs=azure-shell

**Comment by jianawu** [comment]
Yes, sure. I followed these steps and they all worked well. 

By the way, I mean instead of starting the sample rely on ADO Pipeline, if there is a small guidance to help me generate the sample repo locally with several commands, and then I can straightly push my code into somewhere (maybe not my personal project), and then give me a simple guidance of how to configure the pipeline to create resource and start training. 
It would help me better in my case. 

I mean, the quick start in the link above is very good and helped me a lot but if I got a way of starting up not totally relying on the ADO Pipeline well also be very helpful. 

Thank you!
